members/single/plugins.php

```diff
@@ -15,7 +15,9 @@
 
 <?php get_header( 'buddypress' ); ?>
 
-<div id="content" role="main" class="<?php do_action( 'content_class' ); ?>">
+
+
+<div id="content" role="main" class="<?php ( ! is_page() && ! $pagename === 'events' ) ? do_action( 'content_class' ) : do_action( 'no_sidebar_content_classes' ); ?>">
```

Comment:
the logic here can be triggered on the professional interests page as well. Is that intentional?

if we're adding our own action, should 'namespace' it / prefix it with `eypd_`

have you also looked into the value of hooking into the existing `content_class` action?

Reply:
Content_class wasn’t easily hookable as it just prints some text so I decided against it.

Reply:
Shit thought I had added eypd

What’s the URL for pro interests. It was unintentional

Reply:
{domain}/members/{your-name}/professional-interests - can also get there via 'recommend events' button on myEYPD

Reply:
Fixed this for pro-interests. I think it should also fix it for other instanaces as well if there are any. The test is more specific now.